assets/app/game_manager.rb

```diff
@@ -115,7 +115,8 @@ def enter_game(game)
       end
 
       store(:game_data, game_data.merge(loaded: true), skip: true)
-      store(:app_route, hs_url(game, game_data)) unless @app_route.include?(hs_url(game, game_data))
+      game_url = hs_url(game, game_data)
+      `window.location = #{game_url}` unless @app_route.gsub('%20', ' ').include?(game_url)
```

Comment:
can we get rid of all this gsub %20 stuff instead making the title matching easier in general by doing something more fuzzy

Reply:
I think by making `Engine.game_by_title()` do the fuzzy matching for whatever string is passed in, that would work. Do you want that change on this PR or separately?